### PR TITLE
Introduced the feature to upload application on non kiwix organisation.

### LINKED ISSUE #183: Deal with a non Kiwix organisation
Assuming someone wants to benefit from our CI/CD but already owns the app in his organisation:
* How could we upload a custom app to his organisation instead of ours? 
* What kind of things should be changed?

### PR DESCRIPTION
**Pull request by MohitMaliFtechiz**
Fixes #183

* Refactored our CD to upload applications on other organizations (If a secret is found of the application it will upload to their organization. Otherwise, it will upload the application in Kiwix organization).
* Added the documentation in the `README` file for educating users on how they can create and configure the service account with the Play Store.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,8 @@ important fields are:
   run.
 - `upload_bundle` A boolean value. When set to true, it uploads the bundle to the Play Store,
   Otherwise, it uploads the APK.
+- `kiwix_organization` A boolean value. When set to true, it uploads the application in kiwix organization,
```
Comment (kelson42) [inline README.md:62]: Why you need this, you just need to check if there is a dedicated secret with the app tag, and otherwerise fallback to the kiwix json?!

File: SERVICE_ACCOUNT.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+This file gives guidelines for organization how they can create and configure service account
```
Comment (kelson42) [inline SERVICE_ACCOUNT.md:1]: Please put all this in the `README.md` in a section `## Releasing in your organisation`

**Review by kelson42 — CHANGES_REQUESTED** [review]
Yes, but we can do simpler.

File: README.md (hunk 1 above)
Comment (MohitMaliFtechiz) [reply README.md]: Thanks for your suggestion.

File: SERVICE_ACCOUNT.md (hunk 2 above)
Comment (MohitMaliFtechiz) [reply SERVICE_ACCOUNT.md]: @kelson42 Done.

**Review by kelson42 — APPROVED** [review]
LGTM

**Pull request merged**